Makefile

```diff
@@ -1,6 +1,23 @@
+GIT_TAG    = $(shell git describe --tags --abbrev=0 --exact-match 2>/dev/null)
+GIT_BRANCH = $(shell git branch --show-current)
+
+IMAGE_TAG ?= ${GIT_TAG}
+ifeq ($(IMAGE_TAG),)
+	IMAGE_TAG := ${GIT_BRANCH}
+endif
+ifeq ($(IMAGE_TAG),)
+	IMAGE_TAG := latest
+endif
+ifeq ($(IMAGE_TAG),main)
+	IMAGE_TAG := latest
+endif
+
+RELEASE_NAME ?= myproject
```

Comment:
Can we set the release name as move2kube by default

Reply:
done